docs/internal_api.yml

```diff
@@ -433,35 +433,61 @@ definitions:
     description: Per tenant storage settings.
     type: object
     properties:
+      type:
+        type: string
+        enum:
+          - s3
+          - azure
+        description: >-
+          The storage provider type 'azure' Blob storage or AWS 's3' (defaults to s3).
       region:
         type: string
-        description: AWS region.
+        description: >-
+          AWS region (s3 only: required).
       bucket:
         type: string
-        description: Bucket name.
+        description: >-
+          S3 Bucket (Azure: container) name.
       uri:
         type: string
-        description: Endpoint URI.
+        description: >-
+          Bucket/container endpoint URI.
       external_uri:
         type: string
-        description: External Endpoint URI (if differ from uri).
+        description: Public Endpoint URI for presigning URLs (s3 only).
       key:
         type: string
-        description: Access key id (for S3 - AWS_ACCESS_KEY_ID).
+        description: >-
+          Access key identifier (Azure: account name).
```

Comment:
there is a separate property called account_name, but from this description I would assume that if I'm using azure then I should provide account name as a key here

Reply:
I added an alias for it only when using Azure for convenience. Internally, they both map to the same value.